examples/webgl_animation_skinning_morph.html

```diff
@@ -121,17 +121,6 @@
 
 				light.castShadow = true;
 
-				light.shadow.mapSize.width = 1024;
-				light.shadow.mapSize.heigth = 1024;
-
-				var d = 390;
-
-				light.shadow.camera.left = -d;
-				light.shadow.camera.right = d;
-				light.shadow.camera.top = d * 1.5;
-				light.shadow.camera.bottom = -d;
```

Comment:
Doesn't removal of this block leave the shadow camera at its default size of [-5, 5, 5, -5]?